x-pack/legacy/plugins/reporting/types.d.ts

```diff
@@ -119,6 +107,8 @@ export interface JobParams {
   post?: JobParamPostPayload;
   panel?: any; // has to be resolved by the request handler
   visType?: string; // has to be resolved by the request handler
+
+  indexPatternId?: string; // for export_types/csv
```

Comment:
This is another indicator that `JobParams` needs to be different for each `export_type`